### DISCUSSION TITLE
Can't create a SQLite read-only connection

### DISCUSSION THREAD

**Discussion by codebling** [dbeaver/dbeaver#148] [Q&A] [answered]
SQLite connection gives an error if the connection is set to read-only mode. 
`java.sql.SQLException: SQLException: Cannot change read-only flag after establishing a connection. Use SQLiteConfig#setReadOnly and SQLiteConfig.createConnection().`

**Comment by serge-rider** [comment, marked as answer]
SQLite driver doesn't support standard JDBC way to open read-only connections.
Set driver parameter "open_mode" to "1" to make connection read-only.
